bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx107-en.html

```diff
@@ -0,0 +1,66 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_107_en') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+<section class="wnp-content-main mzp-l-content mzp-t-content-lg">
+  <div class="wnp-main-img">
+    {% include 'firefox/whatsnew/includes/fx107/vpn-shopping.svg' %}
+  </div>
+
+  <h2 class="wnp-main-title">Privacy is the gift of the season</h2>
+  <h3 class="wnp-sub-title">Get Mozilla VPN</h3>
+
+  <p class="wnp-main-tagline">
+    With travel logistics, holiday parties and gift ideas — you have too much on
+    your plate to worry about privacy. Mozilla VPN protects your location and
+    browsing activity when you’re online, so you can have peace of mind.
+  </p>
+
+  <p class="wnp-main-cta">
+    {{ vpn_product_referral_link(
+      referral_id='whatsnew-107-en',
+      link_text='Get Mozilla VPN',
+      class_name='mzp-t-product',
+      optional_attributes= {
+          'data-cta-text' : 'Get Mozilla VPN',
+          'data-cta-type' : 'button'
+      })
+    }}
+  </p>
+
+  {% if switch('firefox_whatsnew_107_en_coupon') %}
```

Comment:
that's interesting, I thought these needed kebab casing, but tested locally, it works as expected 💯

Reply:
Either works, I can never remember which is the standard though... I guessed wrong this time.